Fix Switch and MediaPlayback feature map zcl definitions

cjswedes opened the pull request:
#### Issue Being Resolved
* Fixes #22681
* Fixes #22802

#### Change overview
* Add featuremap to MediaPlayback xml definition
* Add cluster link element in SwitchFeature bitmap xml definition

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/project-chip/connectedhomeip?pullRequest=22804) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/project-chip/connectedhomeip?pullRequest=22804) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/project-chip/connectedhomeip?pullRequest=22804) it.</sub>

bzbarsky-apple commented:
> Add featuremap to MediaPlayback xml definition

@cjswedes I don't see that in the PR?

cjswedes commented:
> > Add featuremap to MediaPlayback xml definition
> 
> @cjswedes I don't see that in the PR?

@bzbarsky-apple I updated to add a globalAttribute tag for featuremap, and updated Zap regeneration
